shared/macros-bash.jinja

```diff
@@ -8,7 +8,7 @@ echo "Setting=$varname1" >> config_file
 #}}
 {{%- macro bash_instantiate_variables() -%}}
 {{%- for name in varargs -%}}
-populate {{{ name }}}
+bash-populate {{{ name }}}
```

Comment:
I think that when you use this macro you want to make a bash variable assignment code generated.

For example, when there is this in a bash remediation:

```
{{{ bash_instantiate_variables("var_rekey_limit_size", "var_rekey_limit_time") }}}
```

you want to generate

```
var_rekey_limit_size="(bash-populate var_key_limit_size)"
var_rekey_limit_time="(bash-populate var_rekey_limit_time)"
```

Then, the code in `ssg/build_remediations.py` that would handle `bash-populate` replacement would be the same to the code that handles `ansible-populate`.